domains/aryanpatel.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "aryanpatel142006",
+           "email": "",
+           "discord": "906554993199415366",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.fa_H8QpLZ_OupaUgljIiW3sj6TLUm0JgsLuRUZSKsk2f7FVzSFrUFZ5LRG3DQXkzPYGuN3PqyZcjtAimqJyUz-p8Ab3lgYkXRigP0v2rKBx_ODE_0BceWAU911DDIjpKOY5gKzxeh-RPnamR55senjLrP1Vb0WsT6lDPZIaA1Z5_m-Go7bHvjdjLJl4i0ttKvFgt414ELXRrySrRLq417kLtXebXwKl-jJhUr5j-0hgCS7_klh7vt65KhuWA4XLAAVdwune4G88RLHcOA3_qEwc039FxsJxjUqfHTcPGQZv_dNSckiQovZPmH2hKaJ-NhF1OisYdhtTwWhRS5yzUFA.Fee5fCqO54I7O42Nbs17Vw.zyIGV1sn88_wI6PoC-GgxeuK8Mubdhtb3mRdXU8ck9RImf8f2tWdqisuK7W-DTdE69CPfQ3vZhdKHT8zni_ksuSuiIffnFL9i2sptxQ7ZZGker1B6sRWLekL1vWDlEU-.eHRvZyJxTHtKsWk_k-XcvA"
+        },
+    
+        "record": {
+            "CNAME": "cname.vercel-dns.com."
```

CNAME can not end with `.`
```suggestion
            "CNAME": "cname.vercel-dns.com"
```